javalib/src/main/scala/java/util/package.scala

```diff
@@ -29,7 +31,194 @@ package object util {
   private[util] def defaultOrdering[E]: Ordering[E] = {
     new Ordering[E] {
       def compare(a: E, b: E): Int =
-        a.asInstanceOf[Comparable[E]].compareTo(b)
+        if (a === null) {
+          if (b === null) 0
+          else -1
+        } else {
+          a.asInstanceOf[Comparable[E]].compareTo(b)
+        }
```

Comment:
ordering itself is not symmetric, do you mean that I have to check also the other side?
TreeSet allow ```null``` values.